services/user/CommonSys/common/rpc.d.ts

```diff
@@ -1,98 +1,115 @@
 declare module "common/rpc.mjs" {
-    function getRootDomain(): Promise<string>;
-
-    function siblingUrl(
-        baseUrl?: string | null,
-        service?: string,
-        path?: string
-    ): Promise<string>;
-
-    function initializeApplet(fn: (data: any) => Promise<void>): void;
-
-    function getJson<T = any>(url: string): Promise<T>;
-
-    const MessageTypes = {
-        Action: "Action",
-        Query: "Query",
-        Operation: "Operation",
-        QueryResponse: "QueryResponse",
-        OperationResponse: "OperationResponse",
-        TransactionReceipt: "TransactionReceipt",
-        SetActiveAccount: "SetActiveAccount",
-    };
-
-    declare class AppletId {
-        constructor(appletName: string, subPath?: string);
-
-        name: string;
-        subPath: string;
-        fullPath: string;
-
-        equals: (appletId: AppletId) => boolean;
-        url: () => Promise<string>;
-        static fromFullPath: (fullPath: string) => AppletId;
-        static fromObject: (obj: string) => AppletId;
-    }
-
-    function getCurrentApplet(): Promise<string>;
-
-    function query<Params, Response>(
-        appletId: AppletId,
-        queryName: string,
-        params?: Params
-    ): Promise<Response>;
-
-    function action<ActionParams>(
-        application: string,
-        actionName: string,
-        params: ActionParams,
-        sender?: string
-    ): void;
-
-    function setActiveAccount(account: string): void;
-
-    type Operation = { id: string; exec: (params: any) => Promise<void> };
-
-    type CallbackResponse = {
-        sender: AppletId;
-        response: any;
-        errors: any;
-    };
-
-    function storeCallback(
-        callback: (res: CallbackResponse) => unknown
-    ): number;
-
-    function executeCallback(
-        callbackId: any,
-        response: CallbackResponse
-    ): boolean;
-
-    function getTaposForHeadBlock(baseUrl?: string): Promise<any>;
-
-    function packAndPushSignedTransaction(
-        baseUrl: any,
-        signedTransaction: any
-    ): Promise<any>;
-
-    function setOperations(operations: Operation[]): void;
-    function setQueries(queries: Query[]): void;
-    function operation<Params>(
-        appletId: AppletId,
-        name: string,
-        params?: Params
-    ): Promise<any>;
-    function signTransaction(
-        baseUrl: string,
-        transaction: any,
-        privateKeys?: string[] // TODO: remove optional once we're done with fake-auth
-    ): Promise<any>;
-    function verifyFields(obj: any, fieldNames: any): boolean;
-
-    function postGraphQLGetJson<GqlResponse>(
-        url: string,
-        query: string
-    ): Promise<GqlResponse>;
-
-    function uint8ArrayToHex(data: Uint8Array): string;
+  function getRootDomain(): Promise<string>;
```

Comment:
we haven't already done this formatting of rpc.d.ts? It surprises me that this would be re-formatted by prettier.

Reply:
I ended up dealing with a merge conflict in this file so maybe that did it.... good question...

Reply:
yeah, on my machine, it indents to 4 spaces. So I don't think this is right. I would resave and ensure prettier is producing the 4-space indent

Reply:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/80084763/193919499-8cc252e3-4609-4f6a-a9be-9bd607813809.png">

Reply:
Now indented 4 spaces